cypress/utils/utils.ts

```diff
@@ -827,6 +825,15 @@ export function deleteAllMigrationWaves(currentPage = false): void {
         });
 }
 
+export function deleteAppByName(name: string): void {
```

Comment:
application.delete method does exactly the same . 
Why do we need this in utils ?